Update Governance documentation to reflect Mainnet (#5512)
celo-org/celo-monorepo: packages/docs/celo-codebase/protocol/governance.md

3.  Referendum
4.  Execution

### Overview of Phases

> Each proposal starts on the **Proposal Queue** where it may receive upvotes for promotion to the Approval phase. Proposals are promoted once per day, and must be promoted within 4 weeks. **Approval** lasts 1 days, during which the proposal must be approved by the Approver. Approved proposals are promoted to the Referendum stage. **Referendum** lasts five days, during which owners of Locked Celo vote yes or no on the proposal. Proposals that satisfy the necessary quorum are promoted to the execution phase. **Execution** lasts up to three days, during which anybody may trigger the execution of the proposal. 

Reviewer: ```suggestion
Each proposal starts on the **Proposal Queue** where it may receive upvotes to move forward in the queue relative to other queued proposals.  Three proposals from the top of the queue are dequeued and promoted to the approval stage per day.  Any proposal that remains in the queue for 4 weeks will expire. **Approval** lasts 1 days, during which the proposal must be approved by the Approver. Approved proposals are promoted to the Referendum stage. **Referendum** lasts five days, during which owners of Locked Celo vote yes or no on the proposal. Proposals that satisfy the necessary quorum are promoted to the execution phase. **Execution** lasts up to three days, during which anybody may trigger the execution of the proposal. 
```

### Proposal

> Any user may submit a Proposal to the `Governance` smart contract, along with a small deposit of CELO. This deposit is required to avoid spam proposals, and is refunded to the proposer if the proposal reaches the Approval stage. A Proposal consists of a list of transactions, and a description URL where voters can get more information about the proposal. Transaction data in the proposal includes the destination address, data, and value. If the proposal passes, the included transactions will be executed by the `Governance` contract.

Reviewer: ```suggestion
Any user may submit a Proposal to the `Governance` smart contract, along with a small deposit of CELO. This deposit is required to avoid spam proposals, and is refunded to the proposer if the proposal reaches the Approval stage. A Proposal consists of a list of transactions, and a description URL where voters can get more information about the proposal. It is encouraged that this description URL points to a CGP document in the [celo-org/celo-proposals](https://github.com/celo-org/celo-proposals) repository. Transaction data in the proposal includes the destination address, data, and value. If the proposal passes, the included transactions will be executed by the `Governance` contract.
```

> Submitted proposals are added to the queue of proposals. While a proposal is on this queue, voters may use their [Locked Celo](./proof-of-stake/locked-gold.md) to **upvote** the proposal. Once per day the top three proposals, by weight of the Locked Gold upvoting them, are dequeued and moved into the Approval phase. Note that if there are fewer than three proposals on the queue, all may be dequeued enough if they have no upvotes. If a proposal has been on the queue for for more than 4 weeks, it expires and the deposit is forfeited.

Reviewer: ```suggestion
Submitted proposals are added to the queue of proposals. While a proposal is on this queue, voters may use their [Locked Celo](./proof-of-stake/locked-gold.md) to **upvote** the proposal. Once per day the top three proposals, by weight of the Locked Gold upvoting them, are dequeued and moved into the Approval phase. Note that if there are fewer than three proposals on the queue, all may be dequeued even if they have no upvotes. If a proposal has been on the queue for for more than 4 weeks, it expires and the deposit is forfeited.
```

​
### Upgrade risks
​

> The core contracts define critical behavior of the Celo network such as CELO and Celo Dollar asset management or validator elections and rewards. Malicious or inadvertent contract bugs could compromise user balances or cause potentially harm, irreversible without a blockchain hard fork.

Reviewer: ```suggestion
The core contracts define critical behavior of the Celo network such as CELO and Celo Dollar asset management or validator elections and rewards. Malicious or inadvertent contract bugs could compromise user balances or potentially cause harm, irreversible without a blockchain hard fork.
```
